src/lib/traitify-ui.js

```diff
@@ -2,10 +2,6 @@ import { h, render } from "preact";
 import Main from "../components/main";
 import Promise from 'promise-polyfill';
 
-if (!window.Promise){
```

Comment:
I'm basically trying to keep our stuff within our stuff so we don't change something on the client.